names/module.py

```diff
@@ -0,0 +1,210 @@
+import discord
+from discord.ext import commands
+
+from pie import check, utils, i18n, logger
+from modules.base.admin.database import BaseAdminModule
+from modules.boards.karma.database import KarmaMember
+
+from .database import Price
+
+_ = i18n.Translator("modules/fun").translate
+bot_log = logger.Bot.logger()
+
+
+class Nicknames(commands.Cog):
+    """Pay by karma and I change your name."""
+
+    def __init__(self, bot: commands.Bot):
+        # Check if dependency module is loaded.
+        if not BaseAdminModule.get("boards.karma").enabled:
+            raise Exception(
+                "Unable to load names module due to missing module `boards.karma`."
+            )
+
+        self.bot = bot
+
+    @commands.bot_has_permissions(manage_nicknames=True)
+    @commands.guild_only()
+    @check.acl2(check.ACLevel.EVERYONE)
+    @commands.cooldown(rate=5, per=20.0, type=commands.BucketType.user)
+    @commands.group(name="nickname", aliases=["name"])
+    async def nickname_(self, ctx):
+        """Change your nickname"""
+        await utils.discord.send_help(ctx)
+
+    @commands.guild_only()
+    @check.acl2(check.ACLevel.EVERYONE)
+    @commands.cooldown(rate=5, per=20.0, type=commands.BucketType.user)
+    @nickname_.command(name="prices")
+    async def prices(self, ctx):
+        """Display prices for various services"""
+
+        embed = utils.discord.create_embed(
+            author=ctx.author,
+            title=_(ctx, "Change of nickname"),
+            description=_(
+                ctx,
+                "Values are in karma points. Use commnand **nickname/name** for more info.",
+            ),
+        )
+        price = Price.get(ctx.guild.id)
+        embed.add_field(
+            name=_(ctx, "Set"),
+            value=_(ctx, "Price **{price}**").format(
+                price=price.set_price if price is not None else "Not set"
```

Comment:
`"Not set"` should be `_(ctx, "not set")` instead.

Reply:
Updated, but popie not picking it up as string for replacement.